net.mkiol.Jupii.yaml

```diff
@@ -0,0 +1,447 @@
+app-id: net.mkiol.Jupii
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+rename-icon: jupii
+command: jupii
+build-options:
+  strip: true
+  no-debuginfo: true
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # v4l2 webcams
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-videos
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Tracker1.Steroids # media files index
+  - --talk-name=org.freedesktop.Tracker3.Endpoint # media files index
+  - --talk-name=org.kde.StatusNotifierWatcher # system tray icon
+  - --env=QT_QUICK_CONTROLS_STYLE=org.kde.desktop
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - "*/cmake/*"
+  - "*.a"
+  - "*.la"
+modules:
+  - name: taglib
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_POSITION_INDEPENDENT_CODE=ON
+      - -DBUILD_SHARED_LIBS=OFF
```

Comment:
here and below, why do you disable shared libraries?

Reply:
I prefer static linkage. In this case, without `-DBUILD_SHARED_LIBS=OFF` only shared lib is generated.

Reply:
> I prefer static linkage 

why?

Reply:
No very good reason, just a convenience. I also have scripts to build non-sandboxed packages and static linkage helps in distribution because I don't need to worry about dependencies. I made copy-paste. In case Flatpak, there is no such problem. I can change it if needed.

Reply:
> Dynamic linking is prefered

Got it. I will change it.

Reply:
Done in 04f14a95f1bd7ed2fce0baf64f84f43579269c2c